.gitignore

```diff
@@ -1,4 +1,4 @@
 build/
 install/
 log/
-
+.vscode/
```

Comment:
Used editors depend on users. so `.vscode/` should be written to user's own ignore lists.
Please remove this statement and add it to your `~/.config/git/ignore` .
(And please remember add null statement in all files 👍 )

Reply:
I resolved this one